src/tseda/vpages/ignn.py

```diff
@@ -41,7 +41,6 @@ class GNNHaplotype(View):
 
     individual_id = param.Integer(
         default=None,
-        bounds=(0, None),
```

Comment:
Why has the lower bound been removed? We know that indices are >= 0. Also, I wonder if we should set the upper bound to the (known) number of individuals.

Reply:
Because the lower bound made it so that if a negative number is entered panel automatically showed the tree for index zero. I thought it would be better to get a warning message when negative numbers are entered to avoid confusion.